components/TestimonialLists/index.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div>
+    <Card :testimonials="testimonials" />
+  </div>
+</template>
+
+<script>
+
+export default {
+
```

Comment:
Minor:

inconsistency, you give a line space between `export default` and `data` .